mod/missions/views/default/forms/missions/advanced-search-form.php

```diff
@@ -23,7 +23,7 @@
 		elgg_echo('missions:experience'),
 		elgg_echo('missions:skill'),
 		elgg_echo('missions:portfolio'),
-		elgg_echo('missions:user_department'),
```

Comment:
I wonder if this is something we want to duplicate / override in the missions_collab mod to not potentially affect connex, it's simple enough so might as well?

Reply:
I am not sure if it's working. Would need a second look from you.